nasa-gibs/worldview
# WV-3045 WVS granule layer snapshots

## Description

> Update WV to pass WVS required data for granule snapshots

## How To Test

1. `git checkout WV-3045-wvs-granule`
2. `npm run watch`
3. Open this [link](http://localhost:3000/?v=-194.83679430000677,-65.4807177676953,51.93454726415837,76.32711913810098&ics=true&ici=5&icd=10&l=VIIRS_NOAA20_CorrectedReflectance_BandsM3-I3-M11_Granule(hidden,count=10),VIIRS_NOAA20_CorrectedReflectance_BandsM11-I2-I1_Granule(hidden,count=10),VIIRS_NOAA20_CorrectedReflectance_TrueColor_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_BandsM3-I3-M11_Granule(hidden,count=10),VIIRS_SNPP_CorrectedReflectance_BandsM11-I2-I1_Granule(hidden,count=10),VIIRS_SNPP_CorrectedReflectance_TrueColor_Granule(hidden,count=10),Land_Water_Map&lg=true&t=2024-04-18-T21%3A10%3A00Z)
4. Create snapshots with different granule layers and make sure they come back correct.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
